apps/server/src/requests/events.ts

```diff
@@ -0,0 +1,5 @@
+import { DomainEvent } from '../domain-event';
+
+export class RequestCreatedEvent extends DomainEvent {
```

Comment:
Why? I don't think it is too redundant.
Should the namming + the import be enought?

Reply:
Because the name of the class is used when persisting the event. I'd prefer to store `RequestCreated` rather than `RequestCreatedEvent` for `event.type` in the database. We may add / keep the `Event` suffix everywhere if we trim it before persisting it.